src/averageAges.js

```diff
@@ -15,13 +15,18 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
-}
+  let male = people.filter(person => person.sex === 'm');
+
+  male = century
+    ? male.filter(person => Math.ceil(person.died / 100) === century)
+    : male;
```

Comment:
It would be better, i think
```suggestion
  const male = people.filter(person => century 
    ? Math.ceil(person.died / 100) === century
    : person.sex === 'm');
```

Reply:
it's not working, because if Math.ceil(person.died / 100) === century -  it's not a man